asa/ja-jp/Content/Topics/Adv_Server_Access/docs/about-asa.htm

```diff
@@ -1,36 +1,5 @@
-<!DOCTYPE html>
-<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd" lang="ja-jp" xml:lang="ja-jp" class="_Skins_okta_html5_topnav_nav_poc" data-mc-search-type="NGram" data-mc-help-system-file-name="okta_help.xml" data-mc-path-to-help-system="../../../../" data-mc-has-content-body="True" data-mc-searchable="True" data-mc-toc-path="[%=System.LinkedHeader%]" data-mc-target-type="WebHelp2" data-mc-runtime-file-type="Topic;Default" data-mc-preload-images="false" data-mc-in-preview-mode="false">
-    <head>
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta name="typeofcontent" content="documentation" />
-        <meta name="audience" content="admin workflows" />
-        <meta name="description" content="チーム、グループ、プロジェクト、ユーザー、サービスユーザー、クライアント、サーバーなど、高度なサーバーアクセスコンポーネントについて学びます。" />
-        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><title>アドバンストサーバーアクセスのコンポーネント</title>
-        <meta charset="utf-8" />
-        <meta name="description" content="" />

```

Comment:
Possible a bug in the English version. We have duplicated `<meta name="description" content=...>`